### ISSUE TITLE
Native Javascript (no framework)

### ISSUE THREAD

**Issue by Karric** [NativeScript/visionos-hello-world#1]
Is it possible to not use a JS framework for this?

**Comment by NathanWalker** [comment]
Yes, see this which is just TypeScript only (what is often called plain or vanilla) with no framework:
https://blog.nativescript.org/develop-visionos-apps-with-typescript/

Can be created with:
```
npm install -g nativescript@vision
ns create myapp --vision
```
